conf/fub_curta.config

```diff
@@ -17,6 +16,7 @@ def membership = ['/bin/bash', '-c', 'groups'].execute().text.trim()
 process {
     executor = 'slurm'
     maxRetries = 3
+    stageInMode = 'copy'
```

Comment:
Ooff that seems heavy (duplication of files), is it necessary for the compute.nodes?

Reply:
Yeah I know 😅 sucks...but until this issue gets fixed with the symlink I don't see any other way. It really seemed like the best workaround for our cluster...